web/src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import { ReactElement } from "react";
+
+export const Footer = (): ReactElement => {
+  const isLargeScreen = useMediaQuery(MediaQueries.desktopAndUp);
+  const { Config } = useConfig();
+
+  return (
+    <div
+      className={`${
+        isLargeScreen ? "flex flex-align-center fjb" : ""
+      } margin-x-auto padding-y-4 grid-container-widescreen desktop:padding-x-7 bg-base-darkest text-white`}
+    >
+      <section aria-label="New Jersey Office of Innovation information and services">
+        <div className={`${isLargeScreen ? "" : "flex flex-column flex-align-center"}`}>
+          <a
```

Comment:
[pebble] In order to better match the Figma, it appears you'll need something like this declaration for links in the footer:

```css
text-underline-offset: 0.25rem;
```

(Not sure if that's something that needs to be set for all links across the site or just these links in the footer, but that makes the underline's spacing better match the design.)

Reply:
i'll bring this up in design qa
https://www.pivotaltracker.com/story/show/186490214